tty/imx6ull-uart/imx6ull-uart.c

```diff
@@ -107,16 +247,17 @@ void uart_thr(void *arg)
 
 		switch (msg.type) {
 		case mtOpen:
-			// TODO: set PGID?
+			msg.o.io.err = uart_open(msg.i.openclose.flags);
```

Comment:
shame I needed to read kernel source code to understand how it works (but that's the problem of our API)